frontend/src/pages/WorkloadDetails/WorkloadDetailsPage.tsx

```diff
@@ -40,7 +40,7 @@ type WorkloadDetailsState = {
 
 type ReduxProps = {
   duration: DurationInSeconds;
-  jaegerInfo?: JaegerInfo;
+  jaegerInfo?: TracingInfo;
```

Comment:
If the type is TracingInfo, maybe the variable should be renamed to tracingInfo.